### PR TITLE
feat: Added 7 workflow Files in GitHub Workflows Folder

### LINKED ISSUE #526: [Request]: Noticed Multiple workflows need for Our Project Repository
### Issue Description:
- **close_old_issues.yml:** This file is used for "Closing Old Issues". It automates the process of closing stale issues or sending notifications for review and possible closure of outdated issues.
- **codeql.yml:** This file is used for "Code Quality Testing". It leverages CodeQL analysis rules to perform automated code quality checks.
- **deno.yml:** This file is used for "Deno". Deno is a runtime environment used for running JavaScript and TypeScript code, and this file likely contains the configuration for Deno-related workflows.
- **issues.yml:** This file is used for "Issue Management". It helps in automating the management of issues, such as organizing, labeling, or assigning issues based on predefined rules.
- **lock.yml:** This file is used for "Locking". It is likely used to prevent discussions or information from being modified or altered by locking conversations or threads.
- **prevent_multiple_issues.yml:** This file is used for "Preventing Multiple Issues". It helps in handling multiple issues in a project simultaneously by providing rules for managing and avoiding conflicting or overlapping issues.

### Describe the solution you'd like

Work on those above Issues

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Related Issue

Closes #526 

## Description
- **Added 7 workflow Files in .github/workflows Folder for Future Use.**
- **Whatever Mentioned in My Issue Description except deno.yml File, remained All workflow Files are Added.**
- **Instead of deno.yml File, Additionally added pr_close_greet.yml and issue_close_greet.yml Files. These files are Very Helpful and Nobody Raised these Two.**

## Checklist

- [x] My code adheres to the established style guidelines of the project.
- [ ] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

**Comment by NarendraKoya999** [comment]
Hi @AshAman999 . Please Review on this PR as soon as Possible and Valid Level Label under GSSOC'23. Thank You !

**Comment by AshAman999** [comment]
It will take some time for me to review it @NarendraKoya999 
I am not an expert in github actions  🫤

**Comment by NarendraKoya999** [comment]
Hi @AshAman999 . Any Update on this PR?

File: .github/workflows/pr_close_greet.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          - name: Comment on PR Closure
+            uses: actions/github-script@v4
+            with:
+              github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+              script: >
+                const { owner, repo, number } = context.issue;
+    
+                const commentAuthor = context.payload.pull_request.user.login;
+
+                const commentBody = `Thank you  @${commentAuthor} , for your valuable time and contribution in our Pentesting-and-Hacking-Scripts. \n \n  Hoping to see you soon with another PR again 😇 \n Wishing you all the best for your journey into Open Source🚀
```
Comment (Viveksati5143) [inline .github/workflows/pr_close_greet.yml:20]:
```suggestion
                const commentBody = `Thank you  @${commentAuthor} , for your valuable time and contribution in our `Eduhub-Community`. \n \n  Hoping to see you soon with another PR again 😇 \n Wishing you all the best for your journey into Open Source🚀
```

**Review by Viveksati5143 — CHANGES_REQUESTED** [review]
x

**Pull request closed without merging**